package.json

```diff
@@ -46,7 +57,7 @@
 	"scripts": {
 		"lint": "eslint '**/*.{ts,tsx}'",
 		"fix": "eslint '**/*.{ts,tsx}' --fix",
-		"build": "tsc -p .",
+		"build": "rm -fr dist/* && tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && ./fixup.sh",
```

Comment:
Let's use `rimraf` instead of `rm` to maintain cross OS compatibility.

Reply:
Done